tools/cockpit.spec

```diff
@@ -175,67 +200,121 @@ install -p -m 644 cockpit.pp %{buildroot}%{_datadir}/selinux/targeted/
 %ifnarch x86_64
 rm -rf %{buildroot}/%{_datadir}/%{name}/docker
 %endif
+%if 0%{?suse_version}
+%suse_update_desktop_file -u -r -G 'Cockpit Server Manager' %{name} System Utility
+mv %{buildroot}%{_datadir}/doc/%{name} %{buildroot}%{_docdir}/%{name}/html
+%endif
 
 %files
+%if 0%{?suse_version}
+%defattr(-,root,root)
```

Comment:
Why isn't this the default? Do all SuSE rpms need this line in each %files section?